app/src/features/apiClient/screens/apiClient/components/views/components/Collection/components/CollectionRunnerView/components/RunConfigView/ParseFileModal/Modals/DataFileModalWrapper.tsx

```diff
@@ -128,12 +132,14 @@ export const DataFileModalWrapper: React.FC<PreviewModalProps> = ({ onClose, onF
             secondaryButton: {
               label: "Replace file",
               onClick: () => {
+                trackCollectionRunFileCleared();
```

Comment:
Replace isn't file cleared. cleared event should only be triggered when file is removed